sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -12,11 +12,18 @@
       ></formio>
     </template>
     <template v-slot:footer>
-      <footer-buttons
-        primaryLabel="Approve now"
-        @primaryClick="approveProgram"
-        @secondaryClick="dialogClosed"
-      />
+      <check-a-e-s-t-permission-role
+        :role="Role.InstitutionApproveDeclineProgram"
+      >
+        <template v-slot="{ isReadonly }">
+          <footer-buttons
+            primaryLabel="Approve now"
```

Comment:
If there is no sufficient role the approve-program modal shouldn't come up right? When the button to open this modal is disabled.

Reply:
as discussed in group call. keeping it